File: app/src/main/java/com/lz/framecase/presenter/Main2Contract.java
package com.lz.framecase.presenter;


import com.lz.fram.base.BasePresenter;
import com.lz.fram.base.BaseView;

/**
 * -------- 日期 ---------- 维护人 ------------ 变更内容 --------
 */
public interface Main2Contract {


    interface View extends BaseView {
    }

    interface Presenter extends BasePresenter<View> {
        void getNewLists();
    }


}

File: app/src/main/java/com/lz/framecase/activity/presenter/Main2Contract.kt
package com.lz.framecase.activity.presenter


import com.lz.fram.base.BasePresenter
import com.lz.fram.base.BaseView

/**
 * -------- 日期 ---------- 维护人 ------------ 变更内容 --------
 */
interface Main2Contract {


    interface View : BaseView

    interface Presenter : BasePresenter<View> {
        fun getNewLists()
    }


}
